spec/index.bs

```diff
@@ -9302,7 +9308,8 @@ dictionary GPUVertexAttribute {
             sizeof(|attrib|.{{GPUVertexAttribute/format}}).
         - |attrib|.{{GPUVertexAttribute/shaderLocation}} is &lt;
             |device|.{{GPUObjectBase/[[device]]}}.{{device/[[limits]]}}.{{supported limits/maxVertexAttributes}}.
-    - For every vertex attribute |var| [=statically used=] by |vertexStage|,
+    - Let |entryPoint| be [$get the entry point$]({{GPUShaderStage/VERTEX}}, |vertexStage|). [=Assert=] this is not `null`.
+    - For every vertex attribute |var| [=statically used=] by |entryPoint|,
```

Comment:
Since this is used only once, it could be all one step?

```suggestion
    - Let |entryPoint| be [$get the entry point$]({{GPUShaderStage/VERTEX}}, |vertexStage|). [=Assert=] it is not `null`.
        For every vertex attribute |var| [=statically used=] by |entryPoint|,
```